enterprise/dev/ci/internal/ci/config.go

```diff
@@ -192,12 +192,16 @@ type MessageFlags struct {
 	// SkipHashCompare, if true, tells buildkite to disable skipping of steps that compare
 	// hash output.
 	SkipHashCompare bool
+
+	//Force run the Chromatic step to run. This allows a user to run the job without marking their PR as ready for review
+	ForceRunStepChromatic bool
 }
 
 // parseMessageFlags gets MessageFlags from the given commit message.
 func parseMessageFlags(msg string) MessageFlags {
 	return MessageFlags{
-		ProfilingEnabled: strings.Contains(msg, "[buildkite-enable-profiling]"),
-		SkipHashCompare:  strings.Contains(msg, "[skip-hash-compare]"),
+		ProfilingEnabled:      strings.Contains(msg, "[buildkite-enable-profiling]"),
+		SkipHashCompare:       strings.Contains(msg, "[skip-hash-compare]"),
+		ForceRunStepChromatic: strings.Contains(msg, "[force-run-step-chromatic]"),
```

Comment:
I think this should target the ready for review condition, rather than chromatic condition, as outlined in https://github.com/sourcegraph/sourcegraph/issues/30625

Reply:
So the message should be:
```suggestion
		ReadyForReview: strings.Contains(msg, "[force-ready-for-review]"),
```

Reply:
And in the code, the condition should proc the `bk.IfReadyForReview()` condition

Reply:
Ah I missed that! Thanks

Reply:
This aligns with the other message flags above, i.e. rather than `force-executor-build` we have `skip-hash-compare`. Having the semantics of message flags target the "broadest" concept means that it continues to make sense regardless of the pipeline generated - `force-run-step-chromatic` seems to imply that it will attach the chromatic step to _any_ build, which we don't want to do :)